cfgov/apache/conf.d/headers.conf

```diff
@@ -0,0 +1,30 @@
+ServerTokens ProductOnly
+ServerSignature Off
+TraceEnable off
+LimitRequestFieldSize 8190
+KeepAlive On
+MaxKeepAliveRequests 500
+
+
+Header always set X-Frame-Options SAMEORIGIN
+Header always set X-XSS-Protection: "1; mode=block"
+Header always set X-Content-Type-Options: nosniff
+
+<LocationMatch "^\/owning-a-home\/.+.pdf">
+        Header set Content-Disposition attachment
+</LocationMatch>
+
+<LocationMatch ^\/static\/.+\.[0-9a-f]{12}\..+$>
+        ExpiresActive On
+        Header always set Edge-Control: cache-maxage=1000d,downstream-ttl=1000d
+        ExpiresDefault "now plus 1000 days"
+</LocationMatch>
+
+# this was a test, that we may want to refer back to later. Leaving it (but commented) for posterity
+#<LocationMatch ^\/company-signup\/$>
+#        Header always set Edge-Control: no-store
+#</LocationMatch>
+
+<LocationMatch ^\/consumer-tools/prepaid-cards\/>
+        Header always set Edge-Control: no-store
+</LocationMatch>
```

Comment:
This doesn't include the Access-Control-Allow-Origin header currently defined in the private headers.conf. Currently we set that to https://www.consumerfinance.gov on production and beta -- which, frankly, seems wrong, which makes me think something isn't configured right there:

```sh
$ curl -I https://beta.consumerfinance.gov
HTTP/1.1 200 OK
...
Access-Control-Allow-Origin: https://www.consumerfinance.gov
```

How has this not bitten us on beta? What role does this header play?

On the dev servers we instead set the headers like this:

```
Header set Access-Control-Allow-Origin: "*"
Header set Access-Control-Allow-Headers: "X-Requested-With, Content-Type, Access-Control-Allow-Headers"
```

Shouldn't we set these similarly here?

Reply:
That header is being provided in a conditional left in ansible:
https://GHE/cfpb-infra/ansible-consumer-finance/blob/remove-apache/roles/frontend/templates/apache-conditionals.conf.j2#L38-L43